src/Http/Http.Extensions/src/ProblemDetailsJsonContext.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.AspNetCore.Http;
 
 [JsonSerializable(typeof(ProblemDetails))]
 [JsonSerializable(typeof(HttpValidationProblemDetails))]
+// ExtensionData
+[JsonSerializable(typeof(IDictionary<string, object?>))]
```

Comment:
Is this necessary? `ProblemDetails` has a property of type `IDictionary<string, object?`, so it should be generated transitively, shouldn't it?

Reply:
It should, however when a custom converter is defined, none of them are generated. That is one of the reasons I have this https://github.com/dotnet/aspnetcore/issues/44132.

![image](https://user-images.githubusercontent.com/12376385/213055385-12f03559-ea98-4246-b85b-aa893abba49e.png)